internal/controller/pipelinerun_controller.go

```diff
@@ -179,14 +179,15 @@ func (p *PipelineRun) PodSets() []kueue.PodSet {
 // * `kueue.konflux-ci.dev/requests-storage`
 // * `kueue.konflux-ci.dev/requests-ephemeral-storage`
 //
-// If no annotation is matched, this function will default
-// the memory requested to `1Gi`.
+// By default, a resource which indicates that the workload requires 1
+// PipelineRun will be added. This is useful for controlling the number
+// of PipelineRuns that can be executed concurrently.
 //
 // WARNING: Annotations are not validated and a panic will
 // happen if they can not be parsed as `resource.Quantity`.
 func (p *PipelineRun) resourcesRequests() corev1.ResourceList {
 	requests := corev1.ResourceList{
-		corev1.ResourceMemory: resource.MustParse(defaultMemoryRequest),
+		PipelineRunCountResourceKey: resource.MustParse("1"),
```

Comment:
I'd suggest ahering to corev1's standard
```suggestion
		ResourcePipelineRunCount: resource.MustParse("1"),
```